core/src/main/java/nl/nn/adapterframework/filesystem/LocalFileSystem.java

```diff
@@ -109,9 +106,70 @@ public void removeFolder(File f) {
 	public String getDirectory() {
 		return directory;
 	}
+
 	public void setDirectory(String directory) {
 		this.directory = directory;
 	}
 
+	@Override
+	public void renameTo(File f, String destination) throws FileSystemException {
+		File dest;
+
+		dest = new File(destination);
+		if (dest.exists()) {
+			if (isForce)
+				dest.delete();
+			else {
+				throw new FileSystemException("Cannot rename file. Destination file already exits.");
+			}
+		}
+		f.renameTo(dest);
+
+	}
+
+	@Override
+	public long getFileSize(File f, boolean isFolder) throws FileSystemException {
+		return f.length();
+	}
+
+	@Override
+	public String getName(File f) throws FileSystemException {
+		return f.getName();
+	}
+
+	@Override
+	public String getCanonicalName(File f, boolean isFolder) throws FileSystemException {
+		try {
+			return f.getCanonicalPath();
+		} catch (IOException e) {
+			throw new FileSystemException(e);
+		}
+	}
+
+	@Override
+	public Date getModificationTime(File f, boolean isFolder) throws FileSystemException {
+		return new Date(f.lastModified());
+	}
+
+	@Override
+	public void augmentDirectoryInfo(XmlBuilder dirInfo, File f) {
+		dirInfo.addAttribute("name", f.getName());
```

Comment:
Currently, we use augmentDirectoryInfo method in list action. Since we don't have a test class for LocalFileSystemSender, we can leave this method empty. However, when we create test class for the class it will fail listActionTest. It needs to have the same structure as other filesystem classes have because in list action test we need to be able parse count in the same way we do now.